editor/src/components/inspector/simplified-pinning-helpers.tsx

```diff
@@ -231,8 +289,13 @@ function detectConstraintsSetForGroupChild(
   metadata: ElementInstanceMetadataMap,
   allElementProps: AllElementProps,
   target: ElementPath,
-): { horizontal: HorizontalPinRequests | 'mixed'; vertical: VerticalPinRequests | 'mixed' } {
-  const constraints = getSafeGroupChildConstraintsArray(allElementProps, target)
+): DetectedPins {
+  const constraints = getConstraintsIncludingImplicitForElement(
+    metadata,
+    allElementProps,
+    target,
+    'only-explicit-constraints', // TODO make sure to match this in getUpdateResizedGroupChildrenCommands
```

Comment:
There are a lot of TODOs in this PR that are probably actually FIXMEs. Please can you ensure the ones that we actually want to fix in the immediate future are tracked